Source/WebCore/html/HTMLAttachmentElement.cpp

```diff
@@ -227,12 +232,12 @@ class AttachmentSaveEventListener final : public EventListener {
     WeakPtr<HTMLAttachmentElement, WeakPtrImplWithEventTargetData> m_attachment;
 };
 
-void HTMLAttachmentElement::updateSaveButton(const AtomString& eventTypeName)
+void HTMLAttachmentElement::showSaveButton(bool show)
```

Comment:
`toggleSaveButton` ?

Reply:
As discuss, I'll revert to `updateSaveButton`.